torchinductor/utils.py

```diff
@@ -33,6 +35,15 @@ def has_triton():
     try:
         import triton
 
+        if triton is not None:
+            # This is temp fix to hint people upgrad triton after https://github.com/pytorch/torchdynamo/pull/1585
+            triton_compiler_version = triton.runtime.version_key().split("-")[6]
+            if triton_compiler_version != "7929002797455b30efce6e41eddc6b57":
```

Comment:
Requirering a specific version sounds a bit harsh? If you only want to prevent that error, you can just detect that the function takes more arguments now.
Also you can have an env variable to override this check so that devs that know what they're doing can set it to skip this.

Reply:
Switched to checking launch-time exception instead. Should be general enough unless the next time triton upgrade triggers exactly the same error. This can also address Jason's FBCODE concern.